src/app/pages/network/explorer/event/event-detail/event-detail.component.ts

```diff
@@ -73,15 +73,15 @@ export class EventDetailComponent implements OnInit, OnDestroy {
           filter((event) => !Array.isArray(event.attributes)), //* if attributes is an array, it's a rpc response and we can discard it
           map((event) => {
             if (event) {
+              let eventFormatted = { ...event };
```

Comment:
why do we need this new copy of event?

Reply:
It's good practice to copy objects in javascript before manipulating them to avoid side effects and affecting where it is referenced elsewhere.

I was seeing unexpected behavior when I console logged `event` in the filter in line 73
```
filter((event) => {
  console.log(event)
  return !Array.isArray(event.attributes))
}
```